assets/airports/mkjp.json

```diff
@@ -43021,4 +43244,5 @@
             ]
         }
     ]
+    }
```

Comment:
The airport fails to load and I get the feeling this bracket has something to do with it